presto-hive/src/main/java/io/prestosql/plugin/hive/orc/OrcPageSource.java

```diff
@@ -161,7 +140,7 @@ public void close()
     public String toString()
     {
         return toStringHelper(this)
-                .add("hiveColumnIndexes", hiveColumnIndexes)
+                .add("orcDataSource", orcDataSource.getId())
```

Comment:
I don't think this really matters, but column adaptation seems closer to hivecolumnindexes.